### PR TITLE
Youtube main.py file changed for uploading in youtube

### PR DESCRIPTION
**Pull request by kidbit-ui-dev**

### CODE REVIEW

File: youtube_upload/main.py [hunk 1]
```diff
@@ -232,7 +231,7 @@ def main(arguments):
     parser.add_option('', '--tags', dest='tags', type="string",
                       help='Video tags (separated by commas: "tag1, tag2,...")')
     parser.add_option('', '--privacy', dest='privacy', metavar="STRING",
-                      default="public", help='Privacy status (public | unlisted | private)')
+                      default="private", help='Privacy status (public | unlisted | private)')
```
Comment (kidbit-engineering) [inline youtube_upload/main.py:234]: Also need to change recordingDetails.

Comment (kidbit-ui-dev) [reply youtube_upload/main.py:234]: done sir

Comment (kidbit-engineering) [reply youtube_upload/main.py:234]: @kidbit-ui-dev I don't see it as part of the PR.

File: youtube_upload/main.py [hunk 2]
```diff
@@ -232,16 +231,13 @@ def main(arguments):
...
     parser.add_option('', '--publish-at', dest='publish_at', metavar="datetime",
                       default=None, help='Publish date (ISO 8601): YYYY-MM-DDThh:mm:ss.sZ')
     parser.add_option('', '--license', dest='license', metavar="string",
                       choices=('youtube', 'creativeCommon'), default='youtube',
                       help='License for the video, either "youtube" (the default) or "creativeCommon"')
-    parser.add_option('', '--location', dest='location', type="string",
-                      default=None, metavar="latitude=VAL,longitude=VAL[,altitude=VAL]",
-                      help='Video location"')
-    parser.add_option('', '--recording-date', dest='recording_date', metavar="datetime",
+    parser.add_option('', '--recording-date', dest='recording_date', metavar="string",
```
Comment (kidbit-engineering) [inline youtube_upload/main.py]: please use date time over here. This will ensure args are passed correctly.

File: youtube_upload/main.py [hunk 3]
```diff
@@ -151,7 +151,6 @@ def upload_youtube_video(youtube, options, video_path, total_videos, index):
 
         },
         "recordingDetails": {
-            "location": lib.string_to_dict(options.location),
             "recordingDate": options.recording_date,
```
Comment (kidbit-engineering) [inline youtube_upload/main.py]: convert to string over here.

**Pull request merged**